internal/nexodus/nexodus.go

```diff
@@ -815,8 +815,8 @@ func (nx *Nexodus) addToDeviceCache(p public.ModelsDevice) {
 	}
 }
 
-func (ax *Nexodus) reconcileDeviceCache() error {
-	peerMap, resp, err := ax.informer.Execute()
+func (nx *Nexodus) reconcileDeviceCache() error {
```

Comment:
Don't worry about it for this PR, but my preference as a best practice would be to split out all of the renaming into its own commit. That makes the review of the logic changes easier, as the reviewer can look at each commit individually.

Reply:
Hey will keep that in mind in the future, definitely realised it was a mistake when i saw how many changes were made as i went through this